VERSIONING.md

```diff
@@ -1,13 +1,41 @@
-# Versioning
+# Versioning and stability
 
-This document describes the versioning policy for this repository. This policy
-is designed so that the following goal can be achieved:
+The OpenTelemetry Collector SIG produces several artifacts for [a variety of audiences](CONTRIBUTING.md#target-audiences). This document describes the versioning and support policy for these artifacts. These policies are designed so that the following goal can be achieved:
 
-**Users are provided a codebase of value that is stable and secure.**
+**Users are provided software artifacts of value that are stable and secure.**
 
-## Public API expectations
+The policies are divided depending on the artifact's target audience. While an artifact is supported, [critical bugs](docs/release.md#bugfix-release-criteria) and security vulnerabilities MUST be addressed. The main criteria for the length of support for an artifact is  how easy it is for an artifact's target audience to adapt to disruptive changes.
 
-The following public API expectations apply to all modules in opentelemetry-collector and opentelemetry-collector-contrib.
+## End-user focused software artifacts
+
+End-user focused software artifacts are those that are intended to be used by [end-users](CONTRIBUTING.md#end-users) of the OpenTelemetry Collector. These artifacts include 
+- Binary distributions of the OpenTelemetry Collector.
+- Collector components.
+
+These artifacts are versioned according to the [semantic versioning v2.0.0](https://semver.org/) specification.
+
+### General considerations
+
+Binary distributions produced by the Collector SIG contain components and features with varying [levels of stability](README.md#stability-levels). We abide by the following principles to relate the Collector's version to the stability of its components and features:
+
+* The Collector's core framework MUST be stable in order for a Collector distribution to be v1.0.0 or higher.
```

Comment:
Do we have the scope of the core framework defined anywhere? Is this just describing the core framework packages used by the components within the distribution?

Reply:
We have not previously defined it and I am not sure how to best write it. By core framework I mean anything that is not a component that you can plug (so e.g. configuration resolution, service behavior, CLI flags...). Put it another way, the common parts of all OCB builds